src/instructions/mod.rs

```diff
@@ -38,6 +38,7 @@ pub use ping::PingResponse;
 /// Data from or for a specific motor.
 ///
 /// Used by synchronous write commands.
+#[derive(Clone, Copy, Eq, PartialEq, Debug, Default)]
```

Comment:
```suggestion
#[derive(Debug, Clone, Eq, PartialEq)]
```

This is just for my OCD, I would like to have `Debug` first because its the trait that pretty much everything should always implement.